include/alpaka/kernel/Traits.hpp

```diff
@@ -23,6 +23,7 @@
     #include <alpaka/workdiv/Traits.hpp>
 #endif
 
+#include <cstdint>
```

Comment:
This header should not be required here: `std::size_t` is not defined in there. Is it left over from an intermediate state with `uni32_t` in the interface?

Reply:
You are right, fixed.